helm/microgateway/README.md

```diff
@@ -1,145 +1,293 @@
-| `wso2.deployment.wso2microgw.resources.limits.cpu`                          | The maximum amount of CPU that should be allocated for a Pod                              | 2000m                        |
+
+| `wso2.deployment.wso2microgw.dockerRegistry` | Docker registry of the microgateway image | "" |
+
+| `wso2.deployment.wso2microgw.imageName` | Image name for microgateway node | "" |
+
+| `wso2.deployment.wso2microgw.imageTag` | Image tag for microgateway node | "" |
+
+| `wso2.deployment.wso2microgw.replicas` | Number of replicas for microgateway node | 1 |
+
+| `wso2.deployment.wso2microgw.minReadySeconds` | Refer to [doc](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deploymentspec-v1-apps)| 1 75 |
+
+| `wso2.deployment.wso2microgw.strategy.rollingUpdate.maxSurge` | Refer to [doc](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deploymentstrategy-v1-apps) | 1 |
+
+| `wso2.deployment.wso2microgw.strategy.rollingUpdate.maxUnavailable` | Refer to [doc](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deploymentstrategy-v1-apps) | 0 |
+
+| `wso2.deployment.wso2microgw.livenessProbe.initialDelaySeconds` | Initial delay for the live-ness probe for microgateway node | 40 |
+
+| `wso2.deployment.wso2microgw.livenessProbe.periodSeconds` | Period of the live-ness probe for microgateway node | 10 |
+
+| `wso2.deployment.wso2microgw.readinessProbe.initialDelaySeconds` | Initial delay for the readiness probe for microgateway node | 40 |
+
+| `wso2.deployment.wso2microgw.readinessProbe.periodSeconds` | Period of the readiness probe for microgateway node | 10 |
+
+| `wso2.deployment.wso2microgw.imagePullPolicy` | Refer to [doc](https://kubernetes.io/docs/concepts/containers/images#updating-images) | Always |
+
+| `wso2.deployment.wso2microgw.resources.requests.memory` | The minimum amount of memory that should be allocated for a Pod | 1Gi |
+
+| `wso2.deployment.wso2microgw.resources.requests.cpu` | The minimum amount of CPU that should be allocated for a Pod | 2000m |
+
+| `wso2.deployment.wso2microgw.resources.limits.memory` | The maximum amount of memory that should be allocated for a Pod | 2Gi |
+
+| `wso2.deployment.wso2microgw.resources.limits.cpu` | The maximum amount of CPU that should be allocated for a Pod | 2000m |
+
+| `wso2.deployment.wso2microgw.ingress.hostname` | Hostname for Microgateway | am.wso2.com |
+
+| `wso2.deployment.wso2microgw.ingress.annotations` | Ingress resource annotations for Microgateway  | - |
+
+| `wso2.deployment.wso2microgw.config` | Custom deployment configuration file for Microgateway  | - |
+
+  
 
 **Note**: The above mentioned default, minimum resource amounts for running WSO2 API Microgateway are based on its [official documentation](https://docs.wso2.com/display/MG301/Installation+Prerequisites#InstallationPrerequisites-MicrogatewayRuntime).
 
+  
+
 ##### 3. Deploy WSO2 Identity server.
```

Comment:
@chathuraswso2 please correct the product name.

Reply:
Fixed in https://github.com/wso2/kubernetes-microgateway/pull/21/commits/5a7cf5fe5f9fdbad72a8d58b2971310548a0dc4d commit